resources/web/wwi/FloatingRobotWindow.js

```diff
@@ -0,0 +1,131 @@
+export default class FloatingRobotWindow {
```

Comment:
This file doesn't respect our coding style where two spaces instead of four should be used for for indentation. See https://github.com/cyberbotics/webots/wiki/Javascript-Coding-Style/#cs401-use-2-spaces-indentation